### PR TITLE
feature#26branches-edit-tag-color

### PR DESCRIPTION
**Pull request by EduJimenez96**
# Description
 - What?: I've changed the color of the notification color when editing a register
 - Why?: These changes could be of help to differentiate the action
 # Testing
 None
  
  ## Screenshots 
Blue notification
blue_notif

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -18,6 +18,10 @@
     display: none;
 }
 
+.alert-info{
```
Comment (jaimelr) [inline app/assets/stylesheets/application.css:21]: Missing space after CSS class: `.alert-info {`

File: app/views/branches/edit.html.erb [hunk 2]
```diff
@@ -1,6 +1,26 @@
 <div class="jumbotron text-center" style="margin-bottom:0">
 <h1>Update Branch</h1>
 </div>
+
+<p id="info"><%= info %></p>
```
Comment (jaimelr) [inline app/views/branches/edit.html.erb:5]: Is this duplicating the `info` message?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -24,6 +24,13 @@
       </div>
     <%end%>
 
+    <% if info.present? %>
+    <div class="alert alert-primary notif">
+      <button type="button" class="close ml-3" data-dismiss="alert">&times;</button>
+      <%= info %>
+    </div>
```
Comment (jaimelr) [inline app/views/layouts/application.html.erb:31]: I suggest add a tab here.
```
<% if info.present? %>
  <div class...>
    . . .
  </div>
<% end %>
```

**Review by jaimelr — COMMENTED** [review]
Just a couple of non-blocking comments.

**Review by EduardoGHdez — APPROVED** [review]
LGTM

**Pull request merged**